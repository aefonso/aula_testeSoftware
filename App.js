
import { Text, View, TextInput, Button  } from 'react-native';


export default () => {
  return (
    <View>
      <Text> Digite um nÃºmero</Text>
      <TextInput placeholder="Digite aqui"/>
      <Button title='Calcular'/>
    </View>
  )
}