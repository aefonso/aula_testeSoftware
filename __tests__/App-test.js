import renderer from 'react-test-renderer'
import App from '../App'

describe('Teste de Snapshoot', () => {
    it('Criação de snapshoot', () => {
        const inicial = renderer.create(<App/>).toJSON();
        expect(inicial).toMatchSnapshot();
    })
})