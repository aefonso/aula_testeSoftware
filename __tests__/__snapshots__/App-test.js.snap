// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de Snapshoot Criação de snapshoot 1`] = `
<View>
  <Text>
     Digite um número
  </Text>
  <TextInput
    placeholder="Digite aqui"
  />
  <View
    accessibilityRole="button"
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Calcular
      </Text>
    </View>
  </View>
</View>
`;
